cmd/skopeo/copy.go

```diff
@@ -63,8 +65,17 @@ func copyHandler(context *cli.Context) error {
 }
 
 var copyCmd = cli.Command{
-	Name:      "copy",
-	Usage:     "Copy an image from one location to another",
+	Name:  "copy",
+	Usage: "Copy an image (manifest, filesystem layers, signatures) from one location to another",
```

Comment:
… and config, if it exists. Is it really useful to bother the user with the details?